src/app/core/google-recaptcha/google-recaptcha.service.ts

```diff
@@ -86,8 +88,15 @@ export class GoogleRecaptchaService {
         return res.hasSucceeded && res.payload && isNotEmpty(res.payload.values) && res.payload.values[0].toLowerCase() === 'true';
       }),
     );
-    registrationVerification$.pipe(
-      switchMap((registrationVerification: boolean) => registrationVerification ? this.loadRecaptchaProperties() : EMPTY),
+    const feedBackVerification$ =  this.configService.findByPropertyName('feedback.verification.enabled').pipe(
+      take(1),
+      getFirstCompletedRemoteData(),
+      map((res: RemoteData<ConfigurationProperty>) => {
+        return res.hasSucceeded && res.payload && isNotEmpty(res.payload.values) && res.payload.values[0].toLowerCase() === 'true';
+      }),
+    );
+    observableCombineLatest([registrationVerification$,feedBackVerification$]).pipe(
+      switchMap(([registrationVerification ,feedBackVerification]) => (registrationVerification || feedBackVerification) ? this.loadRecaptchaProperties() : EMPTY),
```

Comment:
Can you rewrite this to accomodate future additions? For example:

```typescript
observableCombineLatest([
  registrationVerification$,
  feedBackVerification$,
]).pipe(
  switchMap((verifications) => verifications.some(Boolean) ? this.loadRecaptchaProperties() : EMPTY),
).subscribe();
```

Then it would just be a matter of adding one line to the array, much easier to merge etc.